### PR TITLE
Add Splunk Integration

### PR DESCRIPTION
**Pull request by c-kruse**
Closes https://github.com/sensu/catalog/issues/108

### CODE REVIEW

File: integrations/splunk/splunk-metrics-and-events/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+## Alerts
+
+This integration does not produce any events that should be processed by an alert or incident management [pipeline].
+
+## Reference Documentation
+
+<!-- Please provide links to any relevant reference documentation to help users learn more and/or troubleshoot this integration; specifically including any third-party software documentation. -->
+
+1. This integration uses [Handler Templating][handler-templating] for variable substitution.
+
```
Comment (jspaleta) [inline integrations/splunk/splunk-metrics-and-events/README.md:41]:
```suggestion
2. Splunk Documentation: [Get data from TCP and UDP ports][splunk-network-input]
3. Splunk Documentation: [input.conf TCP port examples][splunk-tcp-input]
```

**Review by jspaleta — CHANGES_REQUESTED** [review]

File: integrations/splunk/splunk-metrics-and-events/sensu-integration.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        api_version: core/v2
+        type: Handler
+        name: splunk-metrics-and-events
+      patches:
+        - path: /spec/socket/host
+          op: replace
+          value: "[[host]]"
+        - path: /spec/socket/port
+          op: replace
+          value: "[[port]]"
```
Comment (jspaleta) [inline integrations/splunk/splunk-metrics-and-events/sensu-integration.yaml:53]: Something is going wrong here. The port value is being patched as a string not an int.. and the handler resource fails to install as a result.
I even copied the handler out of the review UI  to a file and used sensuctl create -f to confirm

**Review by jspaleta — CHANGES_REQUESTED** [review]

File: integrations/splunk/splunk-metrics-and-events/sensu-integration.yaml (hunk 2 above)
Comment (jspaleta) [inline integrations/splunk/splunk-metrics-and-events/sensu-integration.yaml:53]:
```suggestion
          value: port
```

Comment (c-kruse) [reply integrations/splunk/splunk-metrics-and-events/sensu-integration.yaml:53]: Good find!

**Review by jspaleta — APPROVED** [review]

**Pull request merged**